src/test/fuzz/miniscript.cpp

```diff
@@ -124,10 +124,13 @@ struct ParserContext {
         return a < b;
     }
 
-    std::optional<std::string> ToString(const Key& key) const
+    std::string ToString(const Key& key, bool* has_priv_key) const
     {
         auto it = TEST_DATA.dummy_key_idx_map.find(key);
-        if (it == TEST_DATA.dummy_key_idx_map.end()) return {};
+        if (it == TEST_DATA.dummy_key_idx_map.end()) {
```

Comment:
In 49f41141c674f83d0b66262a5b21aad623704264 _descriptor: ToPrivateString() pass if  at least 1 priv key exists_: maybe for good measure:

```cpp
*has_priv_key = false;
```

In both places it seems better to check that `priv_key` exists, even if that's always the case in this fuzzer. IIUC in that case you can leave out `tmp{false}` and just pass `nullptr` below.